fullscreen/model/remove-first-manual.html

```diff
@@ -23,6 +23,9 @@
             assert_equals(document.fullscreenElement, last);
             assert_equals(event.target, last);
             first.remove();
+            // Both /first/ and /last/ were removed from the top layer and
+            // thus the fullscreen element synchronously becomes null.
+            assert_equals(document.fullscreenElement, null);
```

Comment:
Thanks, I think I understand how to spec something similar then. I'll decide what to do before landing the top layer change in Chromium, which is what will affect these tests, or comment back here if I decide on no change. Can you review so the spec and tests come into agreement first?